testcontainers-common/src/main/java/com/playtika/test/common/utils/ContainerUtils.java

```diff
@@ -43,26 +45,43 @@
 import java.util.function.Consumer;
 
 @UtilityClass
-@Slf4j
 public class ContainerUtils {
 
     public static final Duration DEFAULT_CONTAINER_WAIT_DURATION = Duration.ofSeconds(60);
 
-    @SuppressWarnings("rawtypes")
-    public long startAndLogTime(GenericContainer container) {
+    public static GenericContainer<?> configureCommonsAndStart(GenericContainer<?> container,
```

Comment:
You can instead define the following signature:
```
public static <T extends GenericContainer<?>> T configureCommonsAndStart(T container
```
You will not need to cast container for every module.

Reply:
![image](https://user-images.githubusercontent.com/3741358/94286586-310b8c00-ff23-11ea-8858-39010b190da8.png)